AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Support/ModelSyncedPayload.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+struct ModelSyncedPayload {
+    var modelName: String
```

Comment:
Favor immutable properties. This may cause you to have to do some additional work in the initial sync operation as you're tallying the counts, but when we emit the payload to the Hub, it should be immutable.